.github/workflows/test-dfsql.yml

```diff
@@ -0,0 +1,132 @@
+name: Test with df-sql
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_call:
+
+env:
+  GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python {{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - uses: actions/checkout@v2
+        name: Checkout df-sql
+        with:
+          repository: "oceanprotocol/df-sql"
+          path: "df-sql"
+      
+      - name: Create CSV dir
+        run: |
+          mkdir ~/.dfcsv
+
+      - name: Setup df-sql env file
+        run: |
+          mv df-sql/.env.sample df-sql/.env
+
+      - name: Run df-sql
+        working-directory: ${{ github.workspace }}/df-sql
+        run: |
+          docker-compose up &
+
+      - name: Install dependencies
+        working-directory: ${{ github.workspace }}
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt --quiet
+  
+      - name: Create Brownie paths
+        run: |
+          mkdir -p /home/runner/.solcx
+          mkdir -p /home/runner/.vvm
+          mkdir -p /home/runner/.brownie/packages
+
+      - name: Restore Brownie compilers
+        id: restore-brownie-compilers
+        uses: actions/cache/restore@v3
+        with:
+            path: |
+              /home/runner/.solcx
+              /home/runner/.vvm
+              /home/runner/.brownie/packages
+            key: ${{ runner.os }}-brownie-deps
+
+      - name: Set env variables
+        run: |
+          echo "DFTOOL_KEY=${{ secrets.DFTOOL_KEY_MULTISIG }}" >> $GITHUB_ENV
+          echo "ADDRESS_FILE=.github/workflows/data/address.json" >> $GITHUB_ENV
+          echo "WEB3_INFURA_PROJECT_ID=${{ secrets.WEB3_INFURA_PROJECT_ID }}" >> $GITHUB_ENV
+          echo "SECRET_SEED=${{ secrets.SECRET_SEED }}" >> $GITHUB_ENV
+          echo "date=2023-01-20" >> $GITHUB_ENV
```

Comment:
these seem like pretty vague names for environment variables. Can we name them more precisely? E.g. maybe `DF_SQL_TEST_DATE`, `DF_SQL_TEST_NOW`?